libqtile/core/manager.py

```diff
@@ -1862,3 +1870,15 @@ def run_extension(self, extension: _Extension) -> None:
     def fire_user_hook(self, hook_name: str, *args: Any) -> None:
         """Fire a custom hook."""
         hook.fire(f"user_{hook_name}", *args)
+
+    @expose_command()
+    def start_repl_server(self) -> None:
+        create_task(repl_server.start(locals_dict={"qtile": self}, port=self.config.repl_port))
```

Comment:
@tych0 Should we give access to any other objects here, other than the qtile instance?

Reply:
Great. That was my thinking too.